source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -273,66 +277,66 @@ def _get_response(cls, json_response, pt_object_origin, pt_object_destination, f
-        journey.duration = int(geovelo_resp['duration'])
-        datetime, represents_start_fallback = fallback_extremity
-        if represents_start_fallback:
-            journey.departure_date_time = datetime
-            journey.arrival_date_time = datetime + journey.duration
-        else:
-            journey.departure_date_time = datetime - journey.duration
-            journey.arrival_date_time = datetime
-
-        journey.durations.total = journey.duration
-        journey.durations.bike = journey.duration
-
-        journey.distances.bike = int(geovelo_resp['distances']['total'])
-
-        previous_section_endtime = journey.departure_date_time
-        for index, geovelo_section in enumerate(geovelo_resp['sections']):
-            section = journey.sections.add()
-            section.type = response_pb2.STREET_NETWORK
-
-            section.duration = int(geovelo_section['duration'])
-            section.begin_date_time = previous_section_endtime
-            section.end_date_time = section.begin_date_time + section.duration
-            previous_section_endtime = section.end_date_time
-
-            section.id = 'section_{}'.format(index)
-            section.length = int(geovelo_section['details']['distances']['total'])
-
-            if index == 0:
-                section.origin.CopyFrom(pt_object_origin)
-            if index == len(geovelo_resp['sections']) - 1:
-                section.destination.CopyFrom(pt_object_destination)
-
-            section.street_network.duration = section.duration
-            section.street_network.length = section.length
-            section.street_network.mode = response_pb2.Bike
-
-            speed = section.length / section.duration if section.duration != 0 else 0
-
-            for geovelo_instruction in itertools.islice(
-                geovelo_section['details']['instructions'], 1, sys.maxsize
-            ):
-                path_item = section.street_network.path_items.add()
-                path_item.name = geovelo_instruction[1]
-                path_item.length = geovelo_instruction[2]
-                path_item.duration = round(path_item.length / speed) if speed != 0 else 0
-                path_item.direction = map_instructions_direction.get(geovelo_instruction[0], 0)
-
-            shape = decode_polyline(geovelo_resp['sections'][0]['geometry'])
-            for sh in shape:
-                section.street_network.coordinates.add(lon=sh[0], lat=sh[1])
-
-            elevations = geovelo_section.get('details', {}).get('elevations', []) or []
-            for geovelo_elevation in itertools.islice(elevations, 1, sys.maxsize):
-                elevation = section.street_network.elevations.add()
-                elevation.distance_from_start = geovelo_elevation[0]
-                elevation.elevation = geovelo_elevation[1]
-                elevation.geojson_index = geovelo_elevation[2]
+        geovelo_resp = json_response
+        for i in geovelo_resp:
```

Comment:
Maybe more representative name than `i`? Also, renaming `json_response` into `geovelo_resp` is not that useful. Doing something like the following?
```suggestion
        for geovelo_resp in json_response:
```